packages/block-editor/src/components/block-types-list/index.js

```diff
@@ -60,11 +60,12 @@ function BlockTypesList( {
 		/* eslint-disable jsx-a11y/no-redundant-roles */
 		<ul role="list" className="block-editor-block-types-list">
 			{ normalizedItems.map( ( item ) => {
+				const { id, icon, isDisabled, title, role } = item;
```

Comment:
Changes in this file are no longer necessary.

Reply:
These changes are not used now, but there is just a destructure and added `role` attribute support here. It might be used by the Inserter also to support proper arrow navigation ( there is a check for specific role menuitem ).

Do you think I should revert these changes?